File: module7/Collaboration/Python/SendMessageToRoom.py
import requests

url = "https://api.ciscospark.com/v1/messages"

payload = "{\n  \"roomId\": \"Y2lzY29zcGFyazovL3VzL1JPT00vMjljMzA0NTAtY2FjNi0xMWU1LWE3NDgtNTNlYzZjNjcyYWJl\",\n  \"file\": \"https://developer.cisco.com/images/spark/spark-logo.png\",\n  \"text\": \"Welcome to Spark!\"\n}"
headers = {
    'authorization': "Bearer NTIyN2U5NGQtY2JjZi00NGYxLWE0OTMtYzMwYTY3OWQzMDljOGE0YzA3ZTctNjY1",
    'content-type': "application/json",
    'cache-control': "no-cache",
    'postman-token': "428367e9-d44a-8015-079d-a9f9561a2664"
    }

response = requests.request("POST", url, data=payload, headers=headers)

print(response.text)

File: module7/Collaboration/NodeJS/SendMessageToRoom.js
var request = require("request");

var options = { method: 'POST',
  url: 'https://api.ciscospark.com/v1/messages',
  headers: 
   { 'postman-token': 'ecf97bb1-2329-26a4-11ef-fbec4d6f8dff',
     'cache-control': 'no-cache',
     'content-type': 'application/json',
     authorization: 'Bearer <your token>' },
  body: 
   { roomId: '<room ID>',
     file: 'https://developer.cisco.com/images/spark/spark-logo.png',
     text: 'Welcome to Spark!' },
  json: true };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
